src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -21,7 +21,7 @@ const en = ({ isDivorce, formState, partner }: CommonContent) => ({
      A conditional order is a document that says the court does not see any reason why you cannot ${
        isDivorce ? 'get a divorce' : 'end your civil partnership'
      }.`,
-    line3: `You can apply for a conditional order on ${formState?.dueDate}.
+    line3: `You can apply for a conditional order on ${formState?.dueDate?.replace('th', '')}.
```

Comment:
might be best to format the date how you want it instead of replaceing the string incase the format changes in the backend later on.